portal/code/cx-portal-shared-components/src/components/basic/Tabs/Tab.tsx

```diff
@@ -0,0 +1,17 @@
+import MuiTab, { TabProps as MuiTabProps } from '@mui/material/Tab'
+
+interface TabProps extends MuiTabProps {}
+
+export const Tab = ({ ...props }: TabProps) => {
```

Comment:
Du müsstest die Komponenten noch in `src/components/index.tsx` exportieren, damit sie aus der Library exportiert werden.

Reply:
Done